WooCommerce/Classes/ViewRelated/Hub Menu/HubMenuViewModel.swift

```diff
@@ -52,10 +52,14 @@ final class HubMenuViewModel: ObservableObject {
     init(siteID: Int64, navigationController: UINavigationController? = nil, featureFlagService: FeatureFlagService = ServiceLocator.featureFlagService) {
         self.siteID = siteID
         self.navigationController = navigationController
-        menuElements = [.woocommerceAdmin, .viewStore, .reviews]
+        menuElements = [.woocommerceAdmin, .viewStore]
+        if featureFlagService.isFeatureFlagEnabled(.inbox) {
```

Comment:
Are we sure we want to display the Inbox menu before Reviews? Do we have an order to follow as far you know?

Reply:
Yes, the order is based on the design - link and screenshot in https://github.com/woocommerce/woocommerce-ios/issues/6106 🙂